ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/drawing-and-display.tsx

```diff
@@ -51,9 +77,89 @@ export const CesiumDrawings = ({
     map,
   })
 
+  const [drawingModel] =
+    drawingModels.length > 0 ? drawingModels.slice(-1) : [undefined]
+
+  const [isDrawing, setIsDrawing] = useState(false)
```

Comment:
❓ Should this be:``` const [isDrawing, setIsDrawing] = useState<boolean>(false)```

Reply:
We can omit the explicit type here, since TypeScript correctly infers that `isDrawing` is a `boolean` from the initial state value.